docs/FAQ.rst

```diff
@@ -206,23 +206,21 @@ The fix is to remove ``/dev/sgx`` then install ``/dev/isgx``
 For instructions on how to do tis, see "Intel SGX in Hardware Mode" at
 https://github.com/hyperledger/avalon/blob/master/PREREQUISITES.md#sgx
 
-How can I prevent a worker from using old keys when I restart a KME or a Singleton worker?
+How can I make a worker from use old keys when I restart a KME or a Singleton worker?
 ------------------------------------------------------------------------------------------
-By default, workers use the same signing and encryption key pairs (asymmetric keys) across
-restarts to ensure validity of historical work order requests/responses. This is also done
-to ensure a smooth functioning in case of abrupt failures of the worker. Sealed data from
-respective worker enclaves are used to recover keys across restarts.
-However, this default behavior can be overriden if fresh pairs of aforementioned keys are
-desired. For this, it has to be ensured that the sealed data created during previous
-startup of the enclave manager (KME or Singleton) is cleaned up before starting again.
-This has been provisioned in the docker mode already. Prepend ``CLEAN_SEALED_DATA=1`` to
-the docker-compose command. For example -
+By default, workers use fresh set of signing and encryption key pairs (asymmetric keys) across
+restarts. Sealed data from respective worker enclaves are used to recover keys across restarts.
+However, the default behavior does not allow recovery from selaed data. In production, this behavior
```

Comment:
We can remove "In production" since we can't assume which mode will be preferred in production